openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -517,3 +520,17 @@ def _validate_email(email):
         raise AccountEmailInvalid(
             u"Email '{email}' format is not valid".format(email=email)
         )
+
+
+def set_has_profile_image(username, has_profile_image=True):
```

Comment:
Hmm, in my view of the universe api.py provides the system-facing api while DRF views provide user-facing access, which ends up being a subset of api.py in practice, with auth and HTTP stuff layered on top.  Maybe that is a shaky assumption?

In any event, I'm ok with moving the function, but would prefer not to point outside packages at helpers.py, since it seems anti-encapsulation to me.